testdata/logs/HardhatLogs.t.sol

```diff
@@ -21,6 +37,179 @@ contract HardhatLogsTest {
         console.log("testMisc", address(1));
 				console.log("testMisc", 42);
     }
+
+	function testConsoleLog() public view {
+		console.log(testStr);
+	}
+
+	function testLogInt() public view {
+		console.logInt(testInt);
+	}
+
+	function testLogUint() public view {
+		console.logUint(testUint);
+	}
+
+	function testLogString() public view {
+		console.logString(testStr);
+	}
+
+	function testLogBool() public view {
+		console.logBool(testBool);
+	}
+
+	function testLogAddress() public view {
+		console.logAddress(testAddr);
+	}
+
+	function testLogBytes() public view {
+		console.logBytes(testBytes);
+	}
+
+	function testLogBytes1() public view {
+		console.logBytes1(bytes1("1"));
+	}
+
+	function testLogBytes2() public view {
+		console.logBytes2(bytes2("2"));
+	}
```

Comment:
just a thought -- it might be cool to have a single `bytes32` value that you set to some constant in `setUp` and then explicitly convert to `bytes1`, `bytes2`, etc in each of these tests. The string conversion adds an extra layer of complexity and makes the expected output a little bit less clear for a human reader.

Reply:
all feedback greatly appreciated, will take a crack at this